app/views/api/clidocs/casestudy_metagenomics.html.erb

```diff
@@ -1,121 +1,434 @@
+<!-- -->>read1/1|1
+<!-- -->35725 2759 1 1 1 2 515393 201174
+<!-- -->>read1/1|2
+<!-- -->1185650 1185650 140110
+<!-- -->>read1/1|3
+<!-- -->1773 47855 4530 1 1 1 1 1
+<!-- -->>read1/1|1R
+<!-- -->40674 1 1 2759
+<!-- -->>read1/1|2R
+<!-- -->1 940557 1 940615 940615 940615 940615 940615 56 2 1 1 1 1 2 2 940615 940615 940615 940615 940615
+<!-- -->>read1/1|3R
+<!-- -->2 1 1 1 2759 1 1 3916 3916 1760
+<!-- -->>read1/2|1
+<!-- -->
+<!-- -->>read1/2|2
+<!-- -->
+<!-- -->>read1/2|3
+<!-- -->940615 940615 940615 940615 940615 204434 204434 204434 204434 2 204434 204434 204434
+<!-- -->>read1/2|1R
+<!-- -->
+<!-- -->>read1/2|2R
+<!-- -->
+<!-- -->>read1/2|3R
+<!-- -->
+<!-- -->>read2/1|1
+<!-- -->
+<!-- -->>read2/1|2
+<!-- -->
+<!-- -->>read2/1|3
+<!-- -->2 392734 392734 392734 392734
+<!-- -->>read2/1|1R
+<!-- -->379 379
+<!-- -->>read2/1|2R
+<!-- -->
+<!-- -->>read2/1|3R
+<!-- -->
+<!-- -->>read2/2|1
+<!-- -->
+<!-- -->>read2/2|2
+<!-- -->
+<!-- -->>read2/2|3
+<!-- -->
+<!-- -->>read2/2|1R
+<!-- -->
+<!-- -->>read2/2|2R
+<!-- -->33882 4930 4930 392734 1 392734 392734 392734 392734 392734 392734 392734 392734 392733 392733 392733 392733 392734
+<!-- -->>read2/2|3R
+<!-- -->
+<!-- -->>read3/1|1
+<!-- -->
+<!-- -->>read3/1|2
+<!-- -->
+<!-- -->>read3/1|3
+<!-- -->332163 332163 332163 2 1 332163 2 332163 332163 332163 332163 332163 1 332163 332163 332163 332163 332163 332163 332163 332163 332163
+<!-- -->>read3/1|1R
+<!-- -->
+<!-- -->>read3/1|2R
+<!-- -->
+<!-- -->>read3/1|3R
+<!-- -->
+<!-- -->>read3/2|1
+<!-- -->
+<!-- -->>read3/2|2
+<!-- -->
+<!-- -->>read3/2|3
+<!-- -->
+<!-- -->>read3/2|1R
+<!-- -->332163 2 2 332163 332163 332163 332163 332163 332163 332163 1 332163 332163 332163 2 2 1 332163 332163
+<!-- -->>read3/2|2R
+<!-- -->5855 1 1 2 519963 5206
+<!-- -->>read3/2|3R
+<!-- -->
+        </pre>
         <p>
-          This allows us to determine the most abundant species-level identifications in the sample <i>(timing: 0 secs)</i>.
-    </p>
+          Note: the real output would contain newlines between taxa,
+          which have been replaced here by spaces for readability.
+        </p>
         <pre>
-<b>$</b> join -t, -1 1 -2 1 distribution.csv taxons.csv | sort -t, -k2,2nr | grep species | head -n30</pre>
+<!--   --><b>$</b> mv selected-seeds.fa found-kmers.fa # optional step, same format
+        </pre>
+        <p>
+          This algorithm requires spatial information which is provided
+          by supplying the <code>-o</code> flag to the previous command.
+          The <code>taxons.tsv</code> is a file describing the NCBI taxonomic tree.
+        </p>
+      </div>
+    </div>
+
+    <div class="card">
+      <div class="card-title card-title-colored"><h2 class="card-title-text">Aggregation</h2></div>
+      <div class="card-supporting-text">
         <p>
-          The 30 most dominant species are:
-    </p>
+          At this step of the pipeline, we have a list of taxon identifiers per protein read,
```

Comment:
remove `read` since not used for proteins

_[Original comment](https://github.ugent.be/unipept/unipept/pull/642#discussion_r16557) by @bmesuere on Mon Aug 20 2018 at 14:02._